src/remover.js

```diff
@@ -1,7 +1,10 @@
 function deleteColumn(event) {
-  if (columns.length > 1) {
-    let target = event.target.closest(".column")
-    target.remove()
-    updateColumnsState()
+  let target = event.target.closest(".column")
+  target.remove()
```

Comment:
target is too general name. let use `event.target.closest(".column")?.remove()` then